dae-alright/src/components/modal/signup-modal/signup-modal.component.jsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import CustomModal from "../custom-modal/custom-modal.component";
+import "./signup-modal.styles.scss";
+
+const btnStyles = {
```

Comment:
Move this to a css file, give it a class, and use the classname here

Reply:
I tried it and it didn't work out.